src/applications/personalization/profile-2/components/connected-apps/actions/index.js

```diff
@@ -40,13 +41,15 @@ export function loadConnectedApps() {
         const hasConnectedApps = data && deletedApps?.length !== data?.length;
 
         recordEvent({
+          event: 'profile-get-connected-apps-retrieved',
```

Comment:
PR approved! But I wonder if we need the `await` on line 38?

Reply:
Hm, that seems indeed a bit duplicate -- technically we'd also need to update it to use both `try` and `catch`. I'll make sure to make the update when we do the refactor of Profile 2.0